content/docs/vmclarity/getting-started/install-vmclarity/_index.md

```diff
@@ -8,7 +8,7 @@ Install the VMClarity backend on the platform of your choice.
 - [AWS](#aws)
 - [Azure](#azure)
 - [GCP](#gcp)
-- [Docker](#docker)
+- [Docker]({{< relref "/docs/vmclarity/getting-started/deploy-docker/_index.md" >}})
```

Comment:
Shouldn't we remove this link, since this whole page will disappear in the end?

Reply:
I think it would be good to have a generic installation or getting started page pointing to the installation guide for each available platform. Additionally, the "Access VMClarity UI" section is common for AWS and Azure, we could maintain it here